### PR TITLE
Update wavefront hpa adapter deploy

### PR DESCRIPTION
**Pull request by ironhalo**
* Add additional features to the deployment
* Not sure what sensible defaults are for the pod resource, please advise
* I can add a readiness/health check if supported?

### CODE REVIEW

File: wavefront-hpa-adapter/values.yaml [hunk 1]
```diff
@@ -28,3 +28,19 @@ adapter:
...
   #   - query: 'ts(aws.sqs.approximatenumberofmessagesvisible)'
   #     name: aws.sqs.messagesvisible
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
```
Comment (vikramraman) [inline wavefront-hpa-adapter/values.yaml:38]: Thanks for the PR. Let's maybe go with

```
limits:
  cpu: 50m
  memory: 100Mi
requests:
  cpu: 5m
  memory: 20Mi
```

**Comment by vikramraman** [comment]
We do not expose any readiness checks yet.

File: wavefront-hpa-adapter/values.yaml (hunk 1 above)
Comment (ironhalo) [reply wavefront-hpa-adapter/values.yaml]: updated limits/requests

**Comment by vikramraman** [comment]
Appreciate the updates!

**Pull request merged**